src/overlays/actors/ovl_En_Ssh/z_en_ssh.c

```diff
@@ -4,12 +4,54 @@
 
 #define THIS ((EnSsh*)thisx)
 
+#define STATE_STUNNED (1 << 0)
+#define STATE_GROUND_START (1 << 2)
+#define STATE_ATTACKED (1 << 3)
+#define STATE_SPIN (1 << 4)
+
+typedef enum EnSshAnimation {
+    ANIM_UP = 1,
+    ANIM_WAIT,
+    ANIM_LAND,
+    ANIM_DROP
+} EnSshAnimation;
```

Comment:
I believe there are missing values in this enum since the arrays in `EnSsh_SetAnimation` have 7 entries.
Also I suggest having a prefix specific to the actor because defines starting with `ANIM_` are very likely to conflict with the rest of the codebase in the future.

Reply:
There's a lot of extra junk in this code because it was largely copied from `EnSt`. I assume the extra animation entries are from that.

Reply:
I think the enum should be completed if possible, even if they aren't used. You might be able to figure out what they are based on animations and the rest of the data, and otherwise I suggest having unknowns in there.

Reply:
So, I tried playing the animations. 0 and 6 are unused animations that EnSt doesn't use either. Hard to describe, but looks like an animation for being knocked backwards. 5 is just a slightly slower version of 4. EnSt has the same animation parameters but doesn't use them either.